Automattic/newspack-theme
# fix: make captions styles more specific

### All Submissions:


### Changes proposed in this Pull Request:

This small PR fixes some consistency issues between the `<figcaption>` tag and the `.wp-caption-text` class for the purposes of consistency in older migrated content.

WordPress is inserting this CSS, which is making these two selectors behave differently in the theme:

```
:root :where(.wp-block-image figcaption) {
  color: #555;
  font-size: 13px;
  text-align: center;
}
```

This PR overrides that so they display the same.

### How to test the changes in this Pull Request:

1. Apply this PR.
2. Add an image block with a caption, and below that, add a paragraph block with the CSS class `.wp-caption-text`.
3. Compare them on the front-end and in the editor; confirm they use the same colour and same font size. 

![CleanShot 2024-12-12 at 10 01 41](https://github.com/user-attachments/assets/9b36793d-2705-4fbd-93d1-845c156d90bb)

4. Test a couple different themes to make sure the styles are consistent (eg. Katharine should have a little accent; Sacha should be bold, etc):

![CleanShot 2024-12-12 at 10 02 11](https://github.com/user-attachments/assets/afe2e863-945f-4f41-8e7e-199f6e2962ba)

![CleanShot 2024-12-12 at 10 03 08](https://github.com/user-attachments/assets/00a81424-676e-4414-be79-7374eb8b03ac)

6. Try switching the heading font under Customizer > Typography and confirm that both caption-types pick it up.
![CleanShot 2024-12-12 at 10 04 24](https://github.com/user-attachments/assets/3be0f95a-2172-437a-b147-8353c49eb66b)

### Other information:
